pkg/cmd/cli/uninstall/uninstall.go

```diff
@@ -125,31 +128,36 @@ func Run(ctx context.Context, kbClient kbclient.Client, namespace string, waitTo
 	}
 
 	// CRDs
-	veleroLabels := labels.FormatLabels(install.Labels())
-	crdList := apiextv1beta1.CustomResourceDefinitionList{}
-	opts := kbclient.ListOptions{
-		Namespace: namespace,
-		Raw: &metav1.ListOptions{
-			LabelSelector: veleroLabels,
+
+	veleroLabelSelector := labels.SelectorFromSet(install.Labels())
+	opts := []kbclient.DeleteAllOfOption{
+		kbclient.InNamespace(namespace),
+		kbclient.MatchingLabelsSelector{
+			Selector: veleroLabelSelector,
 		},
 	}
-	if err := kbClient.List(context.Background(), &crdList, &opts); err != nil {
-		errs = append(errs, errors.WithStack(err))
-	} else {
-		if len(crdList.Items) == 0 {
-			fmt.Print("Velero CRDs do not exist, skipping.\n")
+	v1CRDsRemoved := false
+	v1crd := &apiextv1.CustomResourceDefinition{}
+	if err := kbClient.DeleteAllOf(ctx, v1crd, opts...); err != nil {
+		if meta.IsNoMatchError(err) {
+			fmt.Println("V1 Velero CRDs not found, skipping...")
```

Comment:
The messages to the user here are not quite the same as they were before. The skip message will only be printed in the case where the API returns an error if the type doesn't exist where previously it would print this if there were no CRDs to delete. I think it's fine as is though.

Reply:
I though about adding positive messages, like "Removing xxx" but would need to add them everywhere.  Maybe later.